scapy/layers/sctp.py

```diff
@@ -280,15 +312,113 @@ class SCTPChunkParamSupportedAddrTypes(_SCTPChunkParam, Packet):
                              4, padwith=b"\x00"), ]
 
 class SCTPChunkParamECNCapable(_SCTPChunkParam, Packet):
-    fields_desc = [ ShortEnumField("type", 32768, sctpchunkparamtypes),
+    fields_desc = [ ShortEnumField("type", 0x8000, sctpchunkparamtypes),
                     ShortField("len", 4), ]
 
+class SCTPChunkParamRandom(_SCTPChunkParam, Packet):
+    fields_desc = [ ShortEnumField("type", 0x8002, sctpchunkparamtypes),
+                    FieldLenField("len", None, length_of="random",
+                                  adjust = lambda pkt,x:x+4),
+                    PadField(StrLenField("random", os.urandom(32),
```

Comment:
I wrote a default value for pure convenience as I didn't needed to manipulate this data for further use.
However I don't feel like an explicit declaration from a user would make much more sense, as the RFC states that "the Random parameter MUST contain a 32-byte Random Number", else "the association MUST be aborted" as a protocol violation.